BUILDING.md

```diff
@@ -170,7 +170,7 @@ Binaries at <https://nodejs.org/download/release/> are produced on:
 | linux-ppc64le           | RHEL 8 with gcc-toolset-10[^6]                                                                              |
 | linux-s390x             | RHEL 8 with gcc-toolset-10[^6]                                                                              |
 | linux-x64               | RHEL 8 with gcc-toolset-10[^6]                                                                              |
-| win-x64 and win-x86     | Windows 2012 R2 (x64) with Visual Studio 2019                                                               |
+| win-x64 and win-x86     | Windows Server 2022 (x64) with Visual Studio 2022                                                               |
```

Comment:
I know this isn't deployed yet, but I've seen the machines in release Jenkins CI